projects/fsastorefrontlib/src/cms-components/my-account/address-book/address-book.module.ts

```diff
@@ -0,0 +1,58 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { NgSelectModule } from '@ng-select/ng-select';
+import {
+  AuthGuard,
+  CmsConfig,
+  I18nModule,
+  provideDefaultConfig,
+  UserAddressService,
+} from '@spartacus/core';
+import {
+  AddressBookComponentService,
+  CardModule,
+  FormErrorsModule,
+  IconModule,
+  SpinnerModule,
+} from '@spartacus/storefront';
+import { FSAddressBookComponent } from './address-book.component';
+import { FSAddressFormComponent } from './address-form/address-form.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    FormsModule,
+    RouterModule,
+    NgSelectModule,
+    IconModule,
+    FormErrorsModule,
+    CardModule,
+    SpinnerModule,
+    I18nModule,
+  ],
+  declarations: [FSAddressBookComponent, FSAddressFormComponent],
+  exports: [FSAddressBookComponent, FSAddressFormComponent],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        AccountAddressBookComponent: {
+          component: FSAddressBookComponent,
+          providers: [
```

Comment:
Check if we need to re-define providers block

Reply:
Yup, providers part is not needed. Fixed!